### PR TITLE
Fixed French translations

### PR DESCRIPTION
**Pull request by Tititesouris**
See my questions for improvements https://github.com/tilkinsc/KiwiItemInfo/issues/20#issuecomment-546682598

### CODE REVIEW

**Comment by tilkinsc** [comment]
Thanks! I'll check it out on my lunch.

File: Locale/enUS/enUS.lua [hunk 1]
```diff
@@ -71,7 +71,7 @@ KiwiItemInfo.Locale["enUS"] = {
 	["COMMAND_HELP12"] = "        * |cFFBBBBBBvalue|r -- either true, false, string, or number",
 	["COMMAND_HELP13"] = "    > |cFF888888aci|r -- toggles alwaysCompareItems CVar",
 	["COMMAND_HELP14"] = "    > |cFF888888search|r -- searches through item database for items",
-	["COMMAND_HELP15"] = "        * |cFFBBBBBB${=,>,<}num|r -- show only ilvls 'num' of operation",
+	["COMMAND_HELP15"] = "        * |cFFBBBBBB${=,>,<}num|r -- show only items of ilvl equal, bigger or smaller than 'num'",
```
Comment (tilkinsc) [inline Locale/enUS/enUS.lua:74]: Does this all fit on one line?

**Review by tilkinsc — CHANGES_REQUESTED** [review]

File: Locale/frFR/frFR.lua [hunk 2]
```diff
@@ -100,7 +100,7 @@ KiwiItemInfo.Locale["frFR"] = {
 	["KIWIII_RELOAD"] = "recharger",
 	["KIWIII_RESET"] = "réinitialiser",
 	["KIWIII_VARS"] = "vars",
-	["KIWIII_SET"] = "set",
+	["KIWIII_SET"] = "affecte",
```
Comment (tilkinsc) [inline Locale/frFR/frFR.lua]: You changed the command. Please reflect the changes in the respective COMMAND_HELP* as well as the usage section in COMMAND_HELP1-3

**Review by tilkinsc — CHANGES_REQUESTED** [review]

**Comment by tilkinsc** [comment]
Just two minor things that you probably didn't realize. Thanks :)

If you don't want to fix them, then I will merge then do it myself later.

File: Locale/enUS/enUS.lua (hunk 1 above)
Comment (Tititesouris) [reply Locale/enUS/enUS.lua:74]: I'm don't know, what's the character limit?

File: Locale/frFR/frFR.lua (hunk 2 above)
Comment (Tititesouris) [reply Locale/frFR/frFR.lua]: Should I keep these in English? It might make more sense as they are fixed command arguments

Comment (Tititesouris) [reply Locale/frFR/frFR.lua]: Or if I translate them, should I keep accented characters? It's very unconventional to have arguments translated in French

Comment (tilkinsc) [reply Locale/frFR/frFR.lua]: Whatever you think is best. Spanish people might not know help means ayuda, for example

File: Locale/enUS/enUS.lua (hunk 1 above)
Comment (tilkinsc) [reply Locale/enUS/enUS.lua:74]: Try and see scenario here.

Comment (Tititesouris) [reply Locale/enUS/enUS.lua:74]: I don't understand what you mean? What is the restriction on the number of characters?

**Pull request merged**